src/main/java/edu/miu/shoppingcartsystem/service/security/WebSecurityConfig.java

```diff
@@ -84,6 +84,18 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         .exceptionHandling().authenticationEntryPoint(unauthorizedHandler).and()
         .sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS).and()
         .authorizeRequests().antMatchers("/api/auth/**").permitAll()
+        .antMatchers("/api/test/**").permitAll()
+
+/*
+            .antMatchers("/v2/api-docs",
+                    "/configuration/ui",
+                    "/swagger-resources/**",
+                    "/configuration/security",
+                    "/swagger-ui/index.html",
+                    "/webjars/**").permitAll()
+*/
+            .antMatchers("/**").permitAll()
```

Comment:
Please remove this line as we are exposing all the end points without any security.